socorro/app/socorro_app.py

```diff
@@ -375,64 +364,3 @@ def setup_metrics(config, local_unused, args_unused):
             raise ValueError('Invalid markus backend "%s"' % backend)
 
     markus.configure(backends=backends)
-
-    return markus.get_metrics('')
-
-
-class App(SocorroApp):
```

Comment:
I folded this into `SocorroApp` and then renamed that as just `App` so we didn't have to make changes across the codebase.